erpnext/selling/doctype/quotation/quotation.js

```diff
@@ -3,6 +3,7 @@
 
 
 {% include 'erpnext/selling/sales_common.js' %}
+{% include 'erpnext/utils' %}
```

Comment:
what are you including here? frappe.provide is different refer https://frappeframework.com/docs/user/en/api/js-utils#frappeprovide, this line is not required.

Reply:
replaced this line by using frappe.provide method